server/room-battle.js

```diff
@@ -973,8 +974,9 @@ class RoomBattle extends RoomGames.RoomGame {
 				name: player.name,
 				avatar: user ? '' + user.avatar : '',
 				team: team,
+				elo: elo,
```

Comment:
I would call this `rating` instead of `elo`. Notice how the entire rest of the codebase calls it `rating`.